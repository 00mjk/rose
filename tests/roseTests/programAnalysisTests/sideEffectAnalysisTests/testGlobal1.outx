Call side effects on project: 
Function  Side Effects main: ::x 
Statement Side Effects testGlobal1.C-4-5-261: ::x 
Call side effects on file: 
Function  Side Effects main: ::x 
Statement Side Effects testGlobal1.C-4-5-261: ::x 
Call side effects within for loop: 
