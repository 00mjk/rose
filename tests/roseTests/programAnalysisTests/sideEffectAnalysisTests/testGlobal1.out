Call side effects on project: 
Function  Side Effects main: ::x 
Statement Side Effects testGlobal1.C-134818144: ::x 
Call side effects on file: 
Function  Side Effects main: ::x 
Statement Side Effects testGlobal1.C-134818144: ::x 
Call side effects within for loop: 
